daemon/execdriver/windows/exec.go

```diff
@@ -114,13 +50,23 @@ func (d *Driver) Exec(c *execdriver.Command, processConfig *execdriver.ProcessCo
 	logrus.Debugln("commandLine: ", createProcessParms.CommandLine)
 
 	// Start the command running in the container.
-	pid, err = hcsshim.CreateProcessInComputeSystem(c.ID, createProcessParms)
-
+	pid, stdin, stdout, stderr, err := hcsshim.CreateProcessInComputeSystem(c.ID, pipes.Stdin != nil, true, !processConfig.Tty, createProcessParms)
 	if err != nil {
 		logrus.Errorf("CreateProcessInComputeSystem() failed %s", err)
 		return -1, err
 	}
 
+	// Now that the process has been launched, begin copying data to and from the named pipes for the std handles.
+	if stdin != nil {
+		stdinCopy(stdin, pipes.Stdin)
+	}
+	if stdout != nil {
+		stdouterrCopy(stdout, "stdout", pipes.Stdout)
```

Comment:
Updating the name is easy enough. Windows and Linux are very different here. This is similar to how Linux (and Windows) both do copies of container IO back to the CLI. In Windows we have a named pipe to communicate to the HCS, hence this a simply a mirror of the previously mentioned copies, but between docker and the HCS.